index.bs

```diff
@@ -1130,6 +1156,392 @@ The following extended attributes are applicable to interfaces:
+Similarly, for [=attributes=], each copy of the accessor property has
+distinct [=built-in function objects=] for its getters and setters.
+
+The order of appearance of an [=includes statement=] does not matter.
+
+No [=extended attributes=] defined in this specification are applicable to [=includes statements=].
+
+<div class="example" id="example-mixin">
+
+    The following [=IDL fragment=] defines an [=interfaces=] and a [=mixin=] stating
+    that the [=mixin=] is always implemented on objects implementing the [=interface=].
+
+    <pre class="webidl">
+        interface Entry {
+          readonly attribute unsigned short entryType;
+          // ...
+        };
+
+        mixin Observable {
+          void addEventListener(DOMString type,
+                                EventListener listener,
+                                boolean useCapture);
+          // ...
+        };
+
+        Entry includes Observable;
+    </pre>
+
+    An ECMAScript implementation would thus have an “addEventListener”
+    property in the prototype chain of every <code class="idl">Entry</code>:
+
+    <pre highlight="js">
+        var e = getEntry();          // Obtain an instance of Entry.
+        typeof e.addEventListener;   // Evaluates to "function".
+    </pre>
+
+</div>
+
+
+<div class="note">
+
+    Mixins allow the sharing of [=attributes=], [=constants=], and [=operations=]
+    across <em>multiple</em> [=interfaces=].
+    If you're only planning to extend a single interface,
+    you might consider using a [=partial interface=] instead.
+
+    For example, instead of:
+
+    <pre class="webidl">
+        mixin WindowSessionStorage {
+          readonly attribute Storage sessionStorage;
+        };
+        Window includes WindowSessionStorage;
+    </pre>
+
+    do:
+
+    <pre class="webidl">
+        partial Window {
+          readonly attribute Storage sessionStorage;
+        };
+    </pre>
+
+    Additionally, you can rely on extending [=mixins=] exposed by other specifications
+    to target common use cases, such as [=exposed|exposing=]
+    a set of [=attributes=], [=constants=], or [=operations=]
+    across both window and worker contexts.
+
+    For example, instead of the common but verbose:
+
+    <pre class="webidl">
+        mixin GlobalCrypto {
+          [Throws] readonly attribute Crypto crypto;
+        };
+
+        Window includes GlobalCrypto;
+        WorkerGlobalScope includes GlobalCrypto;
+    </pre>
+
+    you can extend the {{WindowOrWorkerGlobalScope}} [=mixin=] using a [=partial mixin|partial=]:
+
+    <pre class="webidl">
+        partial mixin WindowOrWorkerGlobalScope {
+          [Throws] readonly attribute Crypto crypto;
+        };
+    </pre>
+</div>
+
+<div data-fill-with="grammar-Partial"></div>
+
+<div data-fill-with="grammar-PartialDefinition"></div>
+
+<pre class="grammar" id="prod-Mixin">
+    Mixin :
+        "mixin" identifier Inheritance "{" MixinMembers "}" ";"
```

Comment:
I don't think we want or need Inheritance, so like Namespace, we can have just Mixin, not PartialMixin.

Reply:
Indeed. Copy/paste error. Thanks for catching it.